content/articles/react-flux-frontend.md

```diff
@@ -0,0 +1,65 @@
+Title: A tour of modern frontend with React and Flux
+Date: 2015-08-18
+Short_summary: React and Flux have swept the frontend in the last year or so. Let's look at it.
+Category: Dev
+Authors: Vincent
+
+
+*Reading time: ~15 minutes.*
+
+In this article we are going to have a look at my preferred set of tools for writing React/Flux webapps.
+If you don't know React or Flux, please look at their respective websites: [React](http://facebook.github.io/react/), [Flux](https://facebook.github.io/flux/).
+<!-- PELICAN_END_SUMMARY -->
+
+
+## Tools of the trade
+The React community is embracing ES6 — and sometimes ES7 — through [Babel](http://babeljs.io/). I personally like Typescript but recently decided to go with Babel for a few of reasons:
+
+- a constant changes in libraries and their sheer number were making it very hard to keep up-to-date definitions
```

Comment:
I guess it's either "a constant change ..." or "constant changes in ...".

I don't know if we should assume the reader knows about "definitions".

Finally according to Tom's point, the bullet points being sentences they also need periods.